examples/mission.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import asyncio
+
+from dronecode_sdk import connect as dronecode_sdk_connect
+from dronecode_sdk import (MissionItem, MissionItems)
+
+drone = dronecode_sdk_connect(host="127.0.0.1")
+
+
+async def run():
+    my_items = []
+    my_items.append(MissionItem(47.398039859999997, 8.5455725400000002, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
+    my_items.append(MissionItem(47.398036222362471, 8.5450146439425509, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
+    my_items.append(MissionItem(47.397825620791885, 8.5450092830163271, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
+
+    await drone.mission.setReturnToLaunchAfterMission(True)
+
+    mission_items = MissionItems(my_items)
+    upload_mission_status, _ = await drone.mission.uploadMission(mission_items)
+    print(f"-- Upload mission: {upload_mission_status}")
+
+    arm_status, _ = await drone.action.arm()
+    print(f"-- Arm result: {arm_status}")
+
+    start_mission_status, _ = await drone.mission.startMission()
+    print(f"-- Start mission: {start_mission_status}")
+
+
+async def print_mission_progress():
+    async for mission_progress in drone.mission.mission_progress():
+        print(f"Mission progress: {mission_progress.current_item_index}/{mission_progress.mission_count}")
+
+
+async def observe_is_in_air():
+    """ Monitors whether the drone is flying or not and
+    returns after landing """
+
+    was_in_air = False
+
+    async for is_in_air in drone.telemetry.in_air():
```

Comment:
We could add some helper functions in the sdk, e.g. `exit_on_land` - what do you think about it?

Reply:
I agree that this is a bit boilerplate, but I'm a bit nervous about adding stuff like that into the SDK :laughing:.

I'd say that for now, we leave it to the user.